structure/0006-governance.md

```diff
@@ -144,10 +153,25 @@ The following members are eligible:
 - All members of all the sub teams of the relevant team
 - All members of all the parent teams of the relevant team
 
+In case the removal is against an administrative board member, all the existing teams are considered as relevant.
+
+### Dismissing a Team
+
+If a team is found to be uncapable of managing itself anymore, any team member or administrative board member can,
+with enough justification, start a vote to remove all team members from the specified team. All organization members
+are eligible to vote.
+
+If the majority votes in favor, all current team members are removed and the administrative board will designate one 
+or many new team leads who will be in charge of building the new team.
```

Comment:
`one or more`, not `one or many`. Also, we capitalize `Admin(istrative) Board` in all our other documents.